### PR TITLE
add/opensearch: generated config docs

### PR DESCRIPTION
**Pull request by laysauchoa**
Add generated config docs

Fix: https://github.com/aiven/devportal/issues/445

### CODE REVIEW

File: docs/products/opensearch/reference/advanced-params.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+List of advanced parameters
+============================
+
+When creating or working with an Aiven for OpenSearch service, several configuration options are available. They are summarized below:
```
Comment (anelook) [inline docs/products/opensearch/reference/advanced-params.rst:4]: Several usually means "more than two but not many.", in our case it feels like way more than several :D Let's rephrase it a bit.

For "When creating or working" would it be more concise to use "When working with". It summarises all scenarios and is shorter

**Review by anelook — CHANGES_REQUESTED** [review]
So happy to see the configuration options being added!
I have same minor comments for all of the commits of this type.

**Comment by laysauchoa** [comment]
Fix comments proposed here: https://github.com/aiven/devportal/pull/440

**Comment by laysauchoa** [comment]
reviewed

**Review by anelook — APPROVED** [review]

**Pull request merged**